rmw_connextdds_common/src/common/rmw_event.cpp

```diff
@@ -64,6 +64,13 @@ rmw_api_connextdds_publisher_event_init(
 }
 
 
+bool
+rmw_event_type_is_supported(rmw_event_type_t rmw_event_type)
```

Comment:
To keep this consistent with how the code is structured, let's make a couple of changes:
- Call this function `rmw_api_connextdds_event_type_is_supported`.
- Add its declaration in the header `rmw_connextdds_common/include/rnw_connextdds/rmw_api_impl.hpp` in the "Event API" section.
- Use the implementation of `is_even_supported` and delete the function.
- Add the function `rmw_event_type_is_supported` in both `rmw_api_impl_ndds.cpp` and `rmw_api_impl_rtime.cpp` in the "Event API" section using the common `rmw_api_connextdds_event_type_is_supported` function.